example.ts

```diff
@@ -0,0 +1,14 @@
+import { Client } from "./index";
```

Comment:
I just thought about it whether we should call it commonClient since you will use it to interact with any API provider :)

Reply:
I personally would stick to `Client` as it is part of the `@commonbase/sdk` and already indicates it is a common client. If you feel `CommonClient` would offer a better understanding of what the client does, I am happy to change it. ☺️